fasterai/filters.py

```diff
@@ -42,8 +43,16 @@ def _model_process(self, orig:PilImage, sz:int)->PilImage:
         x =  pil2tensor(model_image,np.float32)
         x.div_(255)
         x,y = self.norm((x,x), do_x=True)
-        result = self.learn.pred_batch(ds_type=DatasetType.Valid, 
-            batch=(x[None].cuda(),y[None]), reconstruct=True)
+
+        if torch.cuda.is_available():
```

Comment:
You could refactor this to avoid this duplicated code, the only thing that needs to be in the if is the batch creation.